### PR TITLE
Dev intake

### PR DESCRIPTION
**Pull request by Floofyer**
Added the basic functions for intake.

### CODE REVIEW

File: Intake.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.robolancers321.subsystems;
+
+public class Intake extends SubsystemBase{
+
+  public CANSparkMax intakemotor;
```
Comment (Gitter499) [inline Intake.java]: use camel case
e.g. intakeMotor

File: Intake.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+public class Intake extends SubsystemBase{
+
+  public CANSparkMax intakemotor;
+  SparkPIDController IntakePIDController;
+
+  public int kport = 0;
+  public double power = 0;
+  public double kFF = 0;
+
+  public void intake() {
```
Comment (Gitter499) [inline Intake.java]: Constructor should always be upper case (same with the class)

Please run ./gradlew spotlessApply

File: Intake.java
Comment (Gitter499) [inline Intake.java:1]: Lmao remove this pls

File: src/main/java/org/robolancers321/subsystem/Intake.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+public class Intake extends SubsystemBase {
+
+  public CANSparkMax intakemotor;
+  SparkPIDController IntakePIDController;
+
+  public int kport = 0;
+  public double power = 0;
+
+  public void intake() {
```
Comment (Gitter499) [inline src/main/java/org/robolancers321/subsystem/Intake.java]: bug, constructor name should match class name and should always be capital

File: src/main/java/org/robolancers321/subsystem/Intake.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Intake extends SubsystemBase {
+
+  public CANSparkMax intakemotor;
+  SparkPIDController IntakePIDController;
+
+  public int kport = 0;
```
Comment (Gitter499) [inline src/main/java/org/robolancers321/subsystem/Intake.java]: use camel case

kPort

File: src/main/java/org/robolancers321/subsystem/Intake.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Intake extends SubsystemBase {
+
+  public CANSparkMax intakemotor;
```
Comment (Gitter499) [inline src/main/java/org/robolancers321/subsystem/Intake.java]: use camel case

intakeMotor

File: src/main/java/org/robolancers321/subsystem/Intake.java
Comment (Gitter499) [inline src/main/java/org/robolancers321/subsystem/Intake.java:1]: Where is the PID controller used?

**Review by Gitter499 — CHANGES_REQUESTED** [review]

**Comment by mattperls-code** [comment]
closing this PR to merge both intake subsystems into their own branch for testing before dev

**Pull request closed without merging**